crud.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime, timezone
```

Comment:
The import statement for `timezone` is incorrect. It should be `from datetime import timezone` instead of `from datetime import datetime, timezone`.